nuclia/sdk/search.py

```diff
@@ -172,17 +173,16 @@ def ask(
         )
         if ask_response.metadata is not None:
             if ask_response.metadata.timings is not None:
-                result.timings = ask_response.metadata.timings.dict()
+                result.timings = ask_response.metadata.timings.model_dump()
             if ask_response.metadata.tokens is not None:
-                result.tokens = ask_response.metadata.tokens.dict()
+                result.tokens = ask_response.metadata.tokens.model_dump()
         return result
 
     @kb
     def ask_json(
         self,
-        *,
```

Comment:
I don't really mind this change but all other methods have the `*` that forces users to use named params.